src/tactics/Board.js

```diff
@@ -639,6 +639,11 @@ export default class Board {
     tilesContainer.pointertap = event => {
       if (this.locked === true) return;
 
+      let { x, y } = event.target.data;
+      let tile = this.getTile(x, y);
+      if (Tactics.game.selectMode === 'target' && !this.isAdjacentToHighlighted(tile))
```

Comment:
This could be just 
`if (Tactics.game.selectMode === 'target' && !this.isAdjacentToHighlighted(event.target.data))`
which would save the unnecessary `getTile` call.

Reply:
Good point.  Please proceed with this change.